### PR TITLE
PNNL Requests, part 1

### PR DESCRIPTION
**Pull request by jasondegraw**
Additions:
* Conditioned floors above grade
* Unconditioned floors above grade
* Conditioned floors below grade
* Unconditioned floors below grade
* BAS (boolean)
* Landmark status (boolean)
* Benchmark year
* Number of facilities

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -1055,6 +1055,64 @@
 					</xs:simpleContent>
 				</xs:complexType>
 			</xs:element>
+			<xs:element name="ConditionedFloorsAboveGrade" minOccurs="0">
```
Comment (fieldkm) [inline BuildingSync.xsd]: this looks ok

File: BuildingSync.xsd [hunk 2]
```diff
@@ -1055,6 +1055,64 @@
...
+					<xs:simpleContent>
+						<xs:extension base="xs:integer">
+							<xs:attribute ref="auc:Source"/>
+						</xs:extension>
+					</xs:simpleContent>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="BuildingAutomationSystem" type="xs:boolean" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Does the facility include a building automation or management system.</xs:documentation>
```
Comment (fieldkm) [inline BuildingSync.xsd]: Should be "?" instead of "."

File: BuildingSync.xsd [hunk 3]
```diff
@@ -1055,6 +1055,64 @@
...
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="HistoricalLandmark" type="xs:boolean" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Does the facility have historical landmark status (e.g. is the facility listed in the National Register of Historic Places).</xs:documentation>
```
Comment (fieldkm) [inline BuildingSync.xsd]: Another "?"

**Comment by jasondegraw** [comment]
@fieldkm I've fixed those question marks for @nllong

**Comment by jasondegraw** [comment]
@fieldkm We didn't go over this part on Thursday, but the last step here is to push the big green merge button.

**Comment by fieldkm** [comment]
I figured, but it gave me 3 choices, and I wanted to be sure I chose the right one.  I’m thinking it’s the first one in the list, but I’ll verify today so that it can be merged and I can record the SHA #s in the spreadsheet.

**Pull request merged**